tests/func/test_get.py

```diff
@@ -36,6 +36,35 @@ def test_get_repo_dir(repo_dir, erepo):
     trees_equal(src, dst)
 
 
+def test_get_regular_file(erepo):
+    src = os.path.join(erepo.root_dir, "some_file")
+    dst = os.path.join(os.getcwd(), "_imported")
```

Comment:
Hm, why do we need dst to be abs? We are in the cwd already, so why not just address it by a relpath? Also, I suppose you wanted it to be "some_file_imported", right?